userspace/libsinsp/sinsp.cpp

```diff
@@ -1364,12 +1364,36 @@ int32_t sinsp::next(OUT sinsp_evt **puevt)
 					"n_evts:%" PRIu64
 					" n_drops:%" PRIu64
 					" n_drops_buffer:%" PRIu64
+					" n_drops_buffer_clone_fork_enter:%" PRIu64
+					" n_drops_buffer_clone_fork_exit:%" PRIu64
+					" n_drops_buffer_execve_enter:%" PRIu64
+					" n_drops_buffer_execve_exit:%" PRIu64
+					" n_drops_buffer_connect_enter:%" PRIu64
+					" n_drops_buffer_connect_exit:%" PRIu64
+					" n_drops_buffer_open_enter:%" PRIu64
+					" n_drops_buffer_open_exit:%" PRIu64
+					" n_drops_buffer_dir_file_enter:%" PRIu64
+					" n_drops_buffer_dir_file_exit:%" PRIu64
+					" n_drops_buffer_other_interest_enter:%" PRIu64
+					" n_drops_buffer_other_interest_exit:%" PRIu64
```

Comment:
I would love it if you could add them also in the `scap-open` example [here](https://github.com/falcosecurity/libs/blob/master/userspace/libscap/examples/01-open/scap_open.c#L700-L716)

Reply:
Will do :)